runtime/src/main/resources/packaged/configuration.yml

```diff
@@ -93,3 +93,23 @@ connector:
   connection.timeout: 10000
   socket.timeout: 10000
 
+uiAuthConfig:
+  authEnabled: True
+  authnUrl: "https://authn-preprod.nm.flipkart.com"
+  clientId: "http://localhost:9999/admin/fsmview/" # Should be exactly the same as what was used during registration
+  clientSecret: "iyMGakS5EwNGGRynxoT9JbNU20JmYFHphR2u+cjid/+mvMMK"
```

Comment:
The client secret stored in the config/code? that too on the public github could be security hole.?
Let me know is my understanding correct?

Reply:
can this come from config service?

Reply:
Config service will add a overhead, as all of these configs will remains same overtime,and doesn't leverage the config service's change propagation to hosts from the bucket. We will delete this branch from flux-incubator, once a fork is created on internal git.